include/zephyr/dt-bindings/adc/stm32h7_adc.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2023 Linaro Ltd.
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_ADC_STM32H7_ADC_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_ADC_STM32H7_ADC_H_
+
+#include <zephyr/dt-bindings/adc/stm32_adc.h>
+
+#define STM32_ADC_RES_16		STM32_ADC_RES(16, 0, 2)
+#define STM32_ADC_RES_14		STM32_ADC_RES(14, 1, 2)
+#define STM32_ADC_RES_12		STM32_ADC_RES(12, 2, 2)
+#define STM32_ADC_RES_10		STM32_ADC_RES(10, 3, 2)
+
+/* 8-bit resolution has a different value for various MCUs */
+#define STM32_ADC_RES_8_H72x_3x_Ax_Bx	STM32_ADC_RES(8, 7, 2)
+#define STM32_ADC_RES_8_H74x_5x		STM32_ADC_RES(8, 4, 2)
```

Comment:
I would prefer STM32H72x_ADC_RES_8 and STM32H74x_ADC_RES_8

Reply:
Done